addons/web/static/tests/views/graph_view_tests.js

```diff
@@ -3478,4 +3480,41 @@ QUnit.module("Views", (hooks) => {
         ]);
         checkDatasets(assert, graph, "data", { data: [56, 26, 4, 105, 48] });
     });
+
+    QUnit.test("fill_temporal is true by default", async function (assert) {
+        assert.expect(1);
+
+        const graph = await makeView({
+            serverData,
+            type: "graph",
+            resModel: "foo",
+        });
+
+        const { context } = getGraphModelMetaData(graph);
+        assert.strictEqual(
+            context.fill_temporal,
+            true,
+            "By default, all the empty temporal slices should be visible"
+        );
+        // Can't test the fixture cause the fill_temporal feature is not in the mock server.
+    });
+
+    QUnit.test("fill_temporal can be changed thought the context", async function (assert) {
+        assert.expect(1);
+
+        const graph = await makeView({
+            serverData,
+            type: "graph",
+            resModel: "foo",
+            context: { fill_temporal: false },
+        });
+
+        const { context } = getGraphModelMetaData(graph);
+        assert.strictEqual(
+            context.fill_temporal,
+            false,
+            "All the empty temporal slices should not be displayed"
+        );
+        // Can't test the fixture cause the fill_temporal feature is not in the mock server.
```

Comment:
same here, I'd prefer to mock the web_read_group rpc and deep equal the context instead.